### ISSUE TITLE
Incorrect xemu SVG icon

### ISSUE THREAD

**Issue by j8r** [xemu-project/xemu#1181]
### Bug Description

When seeing https://github.com/mborgerson/xemu/blob/master/ui/icons/xemu.svg, we can notice that the `xemu` name is on the right side, going through the circle, instead of being centered.

### Expected Behavior

The xemu icon should look like https://github.com/mborgerson/xemu/blob/master/ui/icons/xemu_512x512.png.

### xemu Version

All

**Comment by dracc** [comment]
Most likely an issue with your SVG renderer.

**Comment by j8r** [comment]
No, because it looks like this on GitHub, locally in my desktop and Android phone.
Have you tried to open the link?

**Comment by j8r** [comment]
The offset is different though. It could be a fixed left padding, which may looks well on some screens but not on others.
I'm not familiar with how SVG works internally so that's just a guess.

**Comment by mborgerson** [comment]
Try opening the icon in Inkscape. Looks fine for me on GH/firefox

**Comment by j8r** [comment]
Same issue
![screen](https://user-images.githubusercontent.com/6562667/178372125-09024659-8654-49c7-bec7-8954b5a3e115.png)
 
 My screen resolution is 1920x1080, may be related.
 
 Also in my phone there is still an offset. but one a bit less pronounced.
![Screenshot_20220712-005622_Firefox](https://user-images.githubusercontent.com/6562667/178372645-2e507ffc-18ac-4116-893a-2e3bcbf18318.png)

**Comment by mborgerson** [comment]
You may need to install the xemu font (ubuntu mono), probably the text should just be converted to a path

**Comment by j8r** [comment]
Right the font is not the same!
An SVG shall be self-contained to properly fix the issue for all cases.
Maybe there is an option in Inkscape to import/embed font characters.

**Comment by GXTX** [comment]
Also broken on my phone.

**Comment by GitYukari** [comment]
The SVG text is referencing a font not installed on users' systems. The font needs to be converted into a shape/path in Illustrator so it renders correctly on any system without the font.
